src/world.js

```diff
@@ -276,6 +279,17 @@ $.extend( $.World.prototype, $.EventSource.prototype, /** @lends OpenSeadragon.W
     },
 
     /**
+     * As a performance optimization, setting this flag to false allows the bounds-change event handler
+     * on tiledImages to skip calls to _figureSizes. If a lot of images are going to be positioned in
+     * rapid succession, this is a good idea. _figuresSizes only needs to be called once when the
+     * positioning is done.
+     * @param {Boolean} [value] The value to which to set autoRefigureSizes.
```

Comment:
Please add a note reminding people to not forget to set it back to true when they're done rearranging, otherwise the system may behave strangely.